h/static/styles/partials-v2/_api-token.scss

```diff
@@ -0,0 +1,11 @@
+.api-token {
+  border: 1px solid $grey-3;
```

Comment:
The style guide doesn't specify this at all, but what I've been starting to do recently is organize properties into two blocks:

```
.selector {
  <properties that affect the box of the element (position, display, width, height etc.)>

  <properties that affect the internal styling of the element (border, background color, color, font-size etc.)>
}
```